src/main/java/kitchenpos/eatinorders/tobe/dto/eatinorder/EatInOrderResponse.java

```diff
@@ -0,0 +1,21 @@
+package kitchenpos.eatinorders.tobe.dto.eatinorder;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+import kitchenpos.eatinorders.tobe.domain.eatinorder.EatInOrder;
+import kitchenpos.eatinorders.tobe.domain.eatinorder.EatInOrderLineItem;
+import kitchenpos.eatinorders.tobe.domain.eatinorder.EatInOrderStatus;
+import kitchenpos.shared.domain.OrderType;
+
+public record EatInOrderResponse(UUID id, OrderType type, EatInOrderStatus status,
+                                 LocalDateTime orderDateTime,
+                                 List<EatInOrderLineItem> eatInOrderLineItems,
+                                 UUID restaurantTableId) {
+
+    public static EatInOrderResponse from(EatInOrder eatInOrder) {
+        return new EatInOrderResponse(eatInOrder.getId(), eatInOrder.getType(),
+            eatInOrder.getStatus(), eatInOrder.getOrderDateTime(), eatInOrder.getOrderLineItems(),
+            eatInOrder.getRestaurantTableId());
```

Comment:
사소하지만 파라미터가 많은 경우 한줄에 하나씩 기술하는게 가독성이 좋을 수 있습니다.
```suggestion
return new EatInOrderResponse(
            eatInOrder.getId(),
            eatInOrder.getType(),
            eatInOrder.getStatus(),
            eatInOrder.getOrderDateTime(),
            eatInOrder.getOrderLineItems(),
            eatInOrder.getRestaurantTableId()
        );
```

Reply:
(전체 수정하였습니다👍👍👍)